ui/src/dashboards/components/VisualizationName.js

```diff
@@ -46,32 +35,21 @@ class VisualizationName extends Component {
   }
 
   render() {
-    const {name} = this.props
-    const {isEditing} = this.state
-    const graphNameClass =
-      name === NEW_DEFAULT_DASHBOARD_CELL.name
-        ? 'graph-name graph-name__untitled'
-        : 'graph-name'
+    const {workingName} = this.state
 
     return (
       <div className="graph-heading">
-        {isEditing ? (
-          <input
-            type="text"
-            className="form-control input-sm"
-            defaultValue={name}
-            onBlur={this.handleInputBlur}
-            onKeyDown={this.handleKeyDown}
-            autoFocus={true}
-            onFocus={this.handleFocus}
-            placeholder="Name this Cell..."
-            spellCheck={false}
-          />
-        ) : (
-          <div className={graphNameClass} onClick={this.handleInputClick}>
-            {name}
-          </div>
-        )}
+        <input
+          type="text"
+          className="form-control input-sm"
+          value={workingName}
```

Comment:
do you need to pass in the workingName as the value? could you instead set the defaultValue to workingName?

Reply:
I switched the input to be controlled, aka using `value` in tandem with `onChange` to update the value